frontend/src/assets/css/overlayfield.css

```diff
@@ -99,4 +99,16 @@
         display: flex;
         padding: 20px;
     }
-}
\ No newline at end of file
+}
+
+.menu-overlay-spinner {
```

Comment:
why not have this in a dedicated menu.css or in already existing header.css?

Reply:
Will add it to header.css. ty